Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -31,13 +31,11 @@ def family
           :penryn
         when 0x6b5a4cd2 # Nehalem
           :nehalem
-        when 0x573B5EEC # Arrandale
```

Comment:
If the same `hw.cpufamily` is used to describe two types of processors, one Intel and one ARM, then we should switch based on `hw.cputype`, rather than “ignore” one type (even if it's an old one). I'd split this into an Intel list and a ARM list.

Reply:
Current list of ARM types in `mach/machine.h` in the 11.0 SDK:

```
#define CPUFAMILY_ARM_9                 0xe73283ae
#define CPUFAMILY_ARM_11                0x8ff620d8
#define CPUFAMILY_ARM_XSCALE            0x53b005f5
#define CPUFAMILY_ARM_12                0xbd1b0ae9
#define CPUFAMILY_ARM_13                0x0cc90e64
#define CPUFAMILY_ARM_14                0x96077ef1
#define CPUFAMILY_ARM_15                0xa8511bca
#define CPUFAMILY_ARM_SWIFT             0x1e2d6381
#define CPUFAMILY_ARM_CYCLONE           0x37a09642
#define CPUFAMILY_ARM_TYPHOON           0x2c91a47e
#define CPUFAMILY_ARM_TWISTER           0x92fb37c8
#define CPUFAMILY_ARM_HURRICANE         0x67ceee93
#define CPUFAMILY_ARM_MONSOON_MISTRAL   0xe81e7ef6
#define CPUFAMILY_ARM_VORTEX_TEMPEST    0x07d34b9f
#define CPUFAMILY_ARM_LIGHTNING_THUNDER 0x462504d2
#define CPUFAMILY_ARM_FIRESTORM_ICESTORM 0x1b588bb3
```

Reply:
Fixed. Also added a few tests, renamed Arrandale to Westmere, and add all ARM-based CPU families the current macOS 11.0 release kernel supports.